### ISSUE TITLE
Displays "failed to send" on SMS message that was sent

### ISSUE THREAD

**Issue by aidan-fitz** [WhisperSystems/Signal-Android#6002]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
When I try to send an SMS message, the app displays "Failed to send" next to the message. However, I got a reply from the person I sent it to, indicating that the message was probably sent. This doesn't seem to happen every time I send a message.

### Steps to reproduce
- send SMS message

**Actual result:** After several minutes, the app displays a "Failed to send" error message
**Expected result:** Message successfully sends and displays a checkmark

### Screenshots

Forgot to take a screenshot after the bug occurred, and it's not happening anymore for some reason.

### Device info

**Device:** Nexus 4
**Android version:** 5.1.1
**Signal version:** 3.25.4

### Link to debug log

I forgot to get the debug log when the bug happened, but if it happens again I'll post it. I've attached the most recent debug log.
https://gist.github.com/anonymous/1af8725d038829032a52454dfc497423

**Comment by smart-fun** [comment]
I had the same issue with a picture that I sent (mms): the message is written as not delivered but the person did receive it. It happened only once so far.

**Comment by riyapenn** [comment]
Delivery report issue?
```
W/SmsSentJob( 6501): Got SMS callback: org.thoughtcrime.securesms.SendReceiveService.SENT_SMS_ACTION , -1
W/PhoneNumberFormatter( 6501): Got local CC: US
W/MessageDatabase( 6501): Updating ID: 2359 to base type: 23

```
Try the current work around of disabling delivery receipts of your messages: Go to Signal > Menu > Settings > SMS and MMS > and Disable SMS delivery receipts

[Related](https://github.com/WhisperSystems/Signal-Android/issues/5895#issuecomment-265080681) to #5895; duplicate of https://github.com/WhisperSystems/Signal-Android/issues/3728?

**Comment by aidan-fitz** [comment]
Yeah, looks like a duplicate of #3728. Thanks for noticing :+1:
